crates/evm/src/executor/inspector/logs.rs

```diff
@@ -82,7 +113,20 @@ const TOPIC: H256 = H256([
 /// Converts a call to Hardhat's `console.log` to a DSTest `log(string)` event.
 fn convert_hh_log_to_event(call: HardhatConsoleCalls) -> Log {
     // Convert the parameters of the call to their string representation using `ConsoleFmt`.
-    let fmt = call.fmt(Default::default());
+    let fmt = foundry_macros::ConsoleFmt::fmt(&call, Default::default());
+    /* NOTE:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+       In the master branch this is call.fmt(Default::default())
+       However, if I change that here, I get the below error.
+       I don't understand why, as the change seems self contained, and if I'm getting an error whereas no such
+       error is generated here, in the master branch ... that implies there's some incompatible diff happeneing
+       elsewhere ... a diff that shouldn't exist! Yikes.
+
+        error[E0277]: the trait bound `&mut std::fmt::Formatter<'_>: std::default::Default` is not satisfied
+       --> crates/evm/src/executor/inspector/logs.rs:117:24
+        |
+    117 |     let fmt = call.fmt(Default::default());
+        |                        ^^^^^^^^^^^^^^^^ the trait `std::default::Default` is not implemented for `&mut std::fmt::Formatter<'_>`
+         */
```

Comment:
This is fine—it's just a warning about an unimplemented trait—if it works like this it's aight!